<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditCustomerInfo</class>
 <widget class="QWidget" name="EditCustomerInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>680</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>380</width>
    <height>680</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>380</width>
    <height>680</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>EditCustomerInfo</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #2c3e50;
    font-family: &quot;Segoe UI&quot;, sans-serif;
    color: #ecf0f1;
}

QLabel {
    font-size: 14px;
}

QLabel#name, QLabel#id, QLabel#Balance, QLabel#account_type,
QLabel#CN, QLabel#AN, QLabel#IBAN, QLabel#CCV2, QLabel#Date {
    font-weight: bold;
    font-size: 16px;
    color: #ffffff;
    background-color: #34495e;
    padding: 6px 10px;
    border-radius: 8px;
}

QGroupBox {
    border: 2px solid #3498db;
    border-radius: 10px;
    margin-top: 10px;
    padding: 10px;
}

QGroupBox::title {
    color: #3498db;
    font-size: 16px;
    font-weight: bold;
}

QLineEdit {
    background-color: #ecf0f1;
    color: #2c3e50;
    padding: 6px;
    border-radius: 6px;
    font-size: 14px;
}

QPushButton {
    background-color: #3498db;
    color: white;
    font-size: 14px;
    padding: 8px 16px;
    border: none;
    border-radius: 8px;
}

QPushButton:hover {
    background-color: #2980b9;
}

QPushButton#logoutButton {
    background-color: #e74c3c;
}

QPushButton#logoutButton:hover {
    background-color: #c0392b;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QLabel" name="LL">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    font-size: 15px;
    font-weight: bold;
    color: rgb(0, 255, 234);
    padding: 10px 0;
    border-bottom: 2px solid #3498db;
    letter-spacing: 1px;
}
</string>
     </property>
     <property name="text">
      <string>Edit Customer Info</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="logoutButton">
     <property name="text">
      <string>Back To Dashboard</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>National ID Number</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="ID_Line"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="search_btn">
          <property name="text">
           <string>Search Customer</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Edit Info</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_10">
           <item>
            <widget class="QLineEdit" name="name_label">
             <property name="font">
              <font>
               <family>Segoe UI</family>
               <pointsize>-1</pointsize>
              </font>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_9">
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>National ID</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="ID_label">
                 <property name="font">
                  <font>
                   <family>Segoe UI</family>
                   <pointsize>-1</pointsize>
                  </font>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignmentFlag::AlignCenter</set>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <item>
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>Age</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="Age_label">
                 <property name="font">
                  <font>
                   <family>Segoe UI</family>
                   <pointsize>-1</pointsize>
                  </font>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignmentFlag::AlignCenter</set>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>Bank Accounts No</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="Accounts_number_label">
                 <property name="font">
                  <font>
                   <family>Segoe UI</family>
                   <pointsize>-1</pointsize>
                  </font>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignmentFlag::AlignCenter</set>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>New Username</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="new_username_lineEdit">
               <property name="placeholderText">
                <string>Leave to not change</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>New Password</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="new_password_lineEdit">
               <property name="echoMode">
                <enum>QLineEdit::EchoMode::Password</enum>
               </property>
               <property name="placeholderText">
                <string>Leave to not change</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Choose Bank Account To Close:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox">
               <item>
                <property name="text">
                 <string>select option...(Leave to not change)</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QPushButton" name="register_btn">
                 <property name="text">
                  <string>register</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="delete_account_btn">
                 <property name="text">
                  <string>Close Customer Account</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
